code/game/machinery/anomaly_pad.dm

```diff
@@ -0,0 +1,125 @@
+///Magical pad that suspends anomalies. Simply apply an anomaly core and it will make the anomaly fly above it
+/obj/machinery/anomaly_pad
+	name = "anomaly pad"
+	desc = "If an anomaly core is applied, this machine will keep it stable and suspended. Looks beautiful and cannot possibly go wrong."
+
+	icon = 'icons/obj/machines/anomaly_pad.dmi'
+	icon_state = "pad"
+
+	circuit = /obj/item/circuitboard/machine/anomaly_pad
+	density = TRUE
+
+	use_power = IDLE_POWER_USE
+	power_channel = EQUIP
+
+	///this is the big beautiful anomaly floating ontop of us
+	var/obj/effect/anomaly/anomaly
+	///our base active power use. Decreases with laser efficiency and is * 0.1 for inactive
+	var/base_power = 500
+
+/obj/machinery/anomaly_pad/RefreshParts()
+	..()
+
+	var/obj/item/stock_parts/micro_laser/L = locate(/obj/item/stock_parts/micro_laser) in contents
+
+	if(L)
+		active_power_usage = base_power / L.rating
+		idle_power_usage = active_power_usage * 0.1
+
+/obj/machinery/anomaly_pad/attackby(obj/item/I, mob/living/user, params)
+
+	if(default_deconstruction_screwdriver(user, icon_state, icon_state, I))
+		update_icon_state()
+		return
+
+	else if(default_deconstruction_crowbar(I))
+		return
+
+	if(!anomaly && istype(I, /obj/item/assembly/signaler/anomaly) && user.dropItemToGround(I) && IsAnomalyApplicable(I, user))
+		CaptureAnomaly(I)
+		playsound(loc, 'sound/machines/click.ogg', 15, TRUE)
+		return
+
+	. = ..()
+
+/obj/machinery/anomaly_pad/process()
+	if(anomaly && (!is_operational() || get_turf(src) != get_turf(anomaly)))
+		ReleaseAnomaly()
+		return FALSE //uuhhh dont know if its ok but im using this to see if the check passed on the subtype
+
+	return TRUE
+
+///Spawn an anomaly from an anomaly core and suspend it ''''''''''safely''''''''''' on us
+/obj/machinery/anomaly_pad/proc/CaptureAnomaly(obj/item/assembly/signaler/anomaly/S)
+	anomaly = S.ReviveAnomaly(get_turf(src))
+	anomaly.Suspend()
+	use_power = ACTIVE_POWER_USE
+
+///let them loose into the world to reign havoc once more
+/obj/machinery/anomaly_pad/proc/ReleaseAnomaly()
+	anomaly.Unsuspend()
+	anomaly = null
+	playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
+	use_power = IDLE_POWER_USE
+
+///Check if an anomaly is applicable. Example is the chemical pad only accepting fluescent anomalies
+/obj/machinery/anomaly_pad/proc/IsAnomalyApplicable(obj/item/assembly/signaler/anomaly/S, mob/living/user)
```

Comment:
```suggestion
/obj/machinery/anomaly_pad/proc/IsAnomalyApplicable(obj/item/I, mob/living/user)
```

Reply:
Object type is checked before IsAnomalyApplicable is called, and I see no reason to downgrade it. The reason it's still checked later is because that needs to specificly be a fluid anomaly.